usaspending_api/database_scripts/job_archive/recompute_idv_idc_types.sql

```diff
@@ -0,0 +1,70 @@
+-- DEV-2226
+-- Expected CLI: psql -v ON_ERROR_STOP=1 -c '\timing' -f recompute_idv_idc_types.sql $DATABASE_URL
+--
+-- Purpose: Recalculate the IDV IDC sub-types (IDV_B_*) using `type_of_idc_description` which is
+--    more populated than `type_of_idc`. Fallback is to store IDV_B as currently.
+--    For generic IDV_B types, store 'Indefinite Delivery Contract' as the `type_description`
```

Comment:
If this is made to update data to reflect a change in business logic due to 2226, could we explicitly mention that ticket in the comments, so future developers can get a sense of what it was being used for?
```suggestion
--    For generic IDV_B types, store 'Indefinite Delivery Contract' as the `type_description`
--
-- Original Use: DEV-2226
```

Reply:
Does the ticket number on line 1 satisfy this?